analysis_driver/dataset.py

```diff
@@ -529,14 +535,18 @@ def _pipeline_instruction(self):
         return instruction
 
     def _is_ready(self):
-        return self.data_threshold and self.pc_q30 > 75 and self._amount_data() > self.data_threshold
+        return self.pc_q30 > 75 and (
+            (self.required_yield_threshold and self._amount_data() > self.required_yield_threshold) or
+            (self.required_coverage_threshold and self._amount_coverage() > self.required_coverage_threshold)
+        )
 
     def report(self):
         runs = query_dict(self.sample, 'aggregated.run_ids')
         non_useable_runs = sorted(set(r[ELEMENT_RUN_NAME] for r in self.non_useable_run_elements))
 
-        s = '%s  (%s / %s  from %s) ' % (
-            super().report(), self._amount_data(), self.data_threshold, ', '.join(runs)
+        s = '%s  (%s / %s and %s / %s from %s) ' % (
```

Comment:
It's not immediately obvious in the report which number is which:

    test_dataset -- this, that, other (1500000000 / 1000000000 and 32 / 30 from a_run, another_run)

I think it would be good to label these numbers, e.g. `yield 1500000000 / 1000000000 and X coverage 32 / 30`, or something similar like adding units.

Reply:
I've added `yield` and `coverage` prefix